config/tipline_strings.yml

```diff
@@ -1237,6 +1237,20 @@ sw:
   subscribed: Kwa sasa umejiunga na jarida letu.
   unsubscribe_button_label: Jiondoe
   unsubscribed: Kwa sasa umejiondoa kutoka jarida letu.
+  privacy_and_purpose: |-
```

Comment:
looks good to me, but can you please add this entry where it belongs with the key alphabetically sorted, between `navigation_button` and `privacy_statement` ?